ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/controllers/AuthController.java

```diff
@@ -1,5 +1,35 @@
 package com.neeraja.ShareMyOffer.controllers;
 
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.neeraja.ShareMyOffer.dto.LoginDTO;
+
+@Controller
+@RequestMapping("/shareMyOffer")
 public class AuthController {
 
+	LoginDTO tempLogin = new LoginDTO();
+	
+	@GetMapping("/login")
+	public String loginPage() {
+		return "login-page";
+	}
+	
+	@PostMapping("/validate")
+	public String validate(@ModelAttribute("login")LoginDTO login,Model model) {
+		tempLogin.setUsername("srivathsa");
```

Comment:
I didn't understand this part. Where are you using the actual login object?

Reply:
Updated!

Reply:
Changed